windows_10.json

```diff
@@ -65,7 +65,9 @@
         "./scripts/chocolatey.bat",
         "./scripts/chocolatey-packages.bat",
         "./scripts/firewall.bat",
-        "./scripts/doit.bat"
+        "./scripts/doit.bat",
+        "./scripts/clean.bat",
+        "./upstream/scripts/compact.bat"
```

Comment:
@avtar and I had some issue with running compact a while ago: it'd be very very slow and only save 200-300MB of the final image. Have you had any issues with it? Sometimes the build time would be >2 hours.